docs/data-sources/user.md

```diff
@@ -47,6 +47,9 @@ The following attributes are exported:
 * `department` - The name for the department in which the user works.
 * `display_name` - The display name of the user.
 * `employee_id` - The employee identifier assigned to the user by the organisation.
+* `employee_type` - (Optional) Captures enterprise worker type. For example, Employee, Contractor, Consultant, or Vendor.
+* `cost_center` - (Optional) The cost center associated with the user.
+* `division` - (Optional) The name of the division in which the user works.
```

Comment:
As these are attributes, we can drop the `(Optional)` prefix
```suggestion
* `employee_type` - Captures enterprise worker type. For example, Employee, Contractor, Consultant, or Vendor.
* `cost_center` - The cost center associated with the user.
* `division` - The name of the division in which the user works.
```

Reply:
done